packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/actions/email/EmailTemplateTest.java

```diff
@@ -0,0 +1,95 @@
+package org.alfresco.rest.actions.email;
+
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+import javax.json.JsonObject;
+
+import org.alfresco.rest.RestTest;
+import org.alfresco.rest.core.JsonBodyGenerator;
+import org.alfresco.rest.core.RestWrapper;
+import org.alfresco.rest.model.RestNodeModel;
+import org.alfresco.utility.model.FolderModel;
+import org.alfresco.utility.model.UserModel;
+import org.alfresco.utility.Utility;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class EmailTemplateTest extends RestTest {
+
+    public static final String MAIL_ACTION = "mail";
+
+    private UserModel adminUser;
+    private UserModel testUser;
+    private FolderModel testFolder;
+
+    @Autowired
+    protected RestWrapper restClient;
+
+    @BeforeClass(alwaysRun = true)
+    public void dataPreparation() throws Exception {
+        adminUser = dataUser.getAdminUser();
+
+        testUser = dataUser.createRandomTestUser();
+        testSite = dataSite.usingUser(testUser)
+                           .createPublicRandomSite();
+        testFolder = dataContent.usingUser(testUser)
+                                .usingSite(testSite)
+                                .createFolder();
+    }
+
+    @Test
+    public void adminCanSendEmailUsingTemplateWithModelAsString() throws Exception
```

Comment:
I can see a test verifying if an admin can send this message, my assumption is that a user shouldn't be able to. Is that a wrong assumption, or is another test case needed?

Btw. I don't know if it's possible for the user to create the .ftl template, in this tests setup the admin seems to create it, but if the user can do it, then possibly creating it with lower priviledges user and testing vs admin or user would be better